lib/index.js

```diff
@@ -94,7 +105,7 @@ class ImageAsset extends BaseAsset {
 
     to_data() { return this._image_data; }
 
-    _get_outgoing_asset_ids() { return []; }
+    _get_outgoing_asset_ids() { return super._get_outgoing_asset_ids(); }
```

Comment:
If the method only has the invocation of `super.same_name_func`, I think you don't need it.